Purchases/Public/RCPurchases.m

```diff
@@ -878,15 +882,13 @@ - (void)handleReceiptPostWithTransaction:(SKPaymentTransaction *)transaction
             [self cachePurchaserInfo:info forAppUserID:self.appUserID];
 
             [self sendUpdatedPurchaserInfoToDelegateIfChanged:info];
-            [self markAttributesAsSynced:subscriberAttributes appUserID:self.appUserID];
```

Comment:
@vegaro is this safe? I'm marking them as synced _before_ caching purchaser info. as long as the appUserID doesn't change, it should be, right?

Reply:
I think that's fine. I am doing the same in Android.